source/include/vx_reference.h

```diff
@@ -283,6 +304,13 @@ vx_size ownSizeOfEnumType(vx_enum item_type);
 void ownReferenceSetScope(vx_reference ref, vx_reference scope);
 
 
+/*! \brief Create reference from a exemplar object
+ * \ingroup group_vx_reference
+ */
+vx_reference ownCreateReferenceFromExemplar(
```

Comment:
I don't see this function definition anywhere, nor is it called from anywhere.

Reply:
removed